### PR TITLE
Feature/drush command

### PR DESCRIPTION
**Pull request by martinyde**
- Adds drush command for creating test.pdf
- Adds label to templates
- Enables definition of return value for elements in pdf

### CODE REVIEW

File: src/Commands/CreatePdf.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Commands\DrushCommands;
+use Drupal\os2forms_digital_post\Manager\TemplateManager;
+use Drupal\Core\Config\ConfigFactoryInterface;
+
+/**
+ * A drush command file.
+ *
+ * @package Drupal\event_database_pull\Commands
```
Comment (rimi-itk) [inline src/Commands/CreatePdf.php:12]: You may want to mention “os2forms_digital_post” here …

File: src/Commands/CreatePdf.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+   * @usage os2forms_digital_post:create_pdf default --submission_id=12345
+   *   Create PDF using default template with data from set submission.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  public function create($template, $options = ['submission_id'=> 0]) {
+    $elements[] = [];
+    $webformLabel = '';
+    if (0 < $options['submission_id']) {
```
Comment (rimi-itk) [inline src/Commands/CreatePdf.php]: This check seems superfluous. You could check that `$options['submission_id']` is actually an integer, but just skipping any checks would be fine, i.e. the code will work and report any invalid submission id.

File: src/Commands/CreatePdf.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    $context = [
+      'label' => $webformLabel,
+      'elements' => $elements,
+      'recipient' => $recipient,
+    ];
+
+    $pathToTemplate = $template;
+    $pdf = $this->templateManager->renderPdf($pathToTemplate, $context);
+    file_put_contents('test.pdf', $pdf);
```
Comment (rimi-itk) [inline src/Commands/CreatePdf.php]: Making it possible the set the output path with an option may be useful. And reporting the output path will be useful, i.e. something like `$this->output()->writeln(sprintf('Pdf written to %s', $outputPath);`.

File: src/Plugin/WebformHandler/DigitalPostWebformHandler.php [hunk 4]
```diff
@@ -314,8 +314,9 @@ public function postSave(WebformSubmissionInterface $webform_submission, $update
       $element = $this->webform->getElement($key);
 
       $elements[] = [
+        'label' => $this->webform->label(),
         'name' => $element['#title'],
-        'value' => $value,
+        'value' => isset($element['#return_value']) ? $element['#return_value'] : $value,
```
Comment (rimi-itk) [inline src/Plugin/WebformHandler/DigitalPostWebformHandler.php]: Write this as `$element['#return_value'] ?? $value`.

File: src/Commands/CreatePdf.php [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+      $webform_submission = $this->entityTypeManager->getStorage('webform_submission')->load($options['submission_id']);
+      if ($webform_submission) {
+        $webform = $webform_submission->getWebform();
+        $webformLabel = $webform->label();
+        $submissionData = $webform_submission->getData();
+        foreach ($submissionData as $key => $value) {
+          $element = $webform->getElement($key, TRUE);
+          $elements[] = [
+            'name' => $element['#title'],
+            'value' => isset($element['#return_value']) ? $element['#return_value'] : $value,
```
Comment (rimi-itk) [inline src/Commands/CreatePdf.php]: Write this as `$element['#return_value'] ?? $value`.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good. Some minor changes and clean-ups needed.

File: src/Commands/CreatePdf.php (hunk 1 above)
Comment (martinyde) [reply src/Commands/CreatePdf.php:12]: Added

File: src/Commands/CreatePdf.php (hunk 2 above)
Comment (martinyde) [reply src/Commands/CreatePdf.php]: REmoved

File: src/Commands/CreatePdf.php (hunk 3 above)
Comment (martinyde) [reply src/Commands/CreatePdf.php]: Added output path and filename options

File: src/Plugin/WebformHandler/DigitalPostWebformHandler.php (hunk 4 above)
Comment (martinyde) [reply src/Plugin/WebformHandler/DigitalPostWebformHandler.php]: Changed

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**